lib/modules/apostrophe-pieces/views/piecesMacros.html

```diff
@@ -77,7 +77,7 @@
 
   #}<div class="apos-modal-filters-toggles">
     {%- for filter in filters.options -%}
-      {{ pills.base( module.filterChoicesToPillChoices(filter), filters.choices[filter.name] ) }}
+      {{ fields.select(filter.name, apos.utils.concat([ { label: filter.label or '—', value: "**ANY**" } ], filter.choices), filters.choices[filter.name]) }}
```

Comment:
`**ANY**` is some sort of magic protected string that's used as a default select option?